packages/react-components/react-textarea/src/stories/TextareaAppearance.stories.tsx

```diff
@@ -9,27 +9,29 @@ const useStyles = makeStyles({
   base: {
     display: 'flex',
     flexDirection: 'column',
+
     '> div': {
       display: 'flex',
       flexDirection: 'column',
       marginTop: tokens.spacingVerticalMNudge,
       ...shorthands.padding(tokens.spacingHorizontalMNudge),
     },
+
     '> div > label': {
       marginBottom: tokens.spacingHorizontalXXS,
       marginLeft: tokens.spacingHorizontalMNudge,
     },
   },
   filledLighter: {
-    backgroundColor: '#8a8a8a',
```

Comment:
We should never use direct colors in our stories as we recommend to use tokens everywhere and show right patterns.

We don't use `tokens` in our examples ➡️ customers don't use `tokens` ➡️  theme switching is broken.

Reply:
I'm okay with this. I'll have the designers who worked on this review as well. This will need to be updated for several other components if we go with this change.

Reply:
👍on the changes from design.

Reply:
I'll file an issue for the rest of the components.

Reply:
nvm! beat me to it 😄